models/Asset/Thumbnail/ImageThumbnailTrait.php

```diff
@@ -429,4 +429,22 @@ private function getFilename(): string
 
         return basename($pathReference['src']);
     }
+
+
+    /**
+     * @description Returns path for thumbnail image in a given file format
+     * @param string $format
+     * @param bool $deferred
+     * @return string
+     */
+    public function getThumbnailPathAs(string $format, $deferred = true)
```

Comment:
@aashan10 I'd really go for `getAsFormat(string $format): self` 
So I think we should return a thumbnail instance again instead of a path, because then you have much more possibilities, such as: 
```php
$asset->getThumbnail('foo')->getAsFormat('avif')->getStream();
// or
$asset->getThumbnail('foo')->getAsFormat('avif')->getMimeType();
```

Reply:
Modification has been made accordingly!